pkg/types/powervs/platform.go

```diff
@@ -0,0 +1,57 @@
+package powervs
+
+// Platform stores all the global configuration that all machinesets
+// use.
+type Platform struct {
+
+	// ServiceInstanceID is the ID of the Power IAAS instance created from the IBM Cloud Catalog
+	ServiceInstanceID string `json:"serviceInstanceID"`
+
+	// PowerVSResourceGroup is the resource group for creating Power VS resources.
+	PowerVSResourceGroup string `json:"powervsResourceGroup"`
+
+	// Region specifies the IBM Cloud colo region where the cluster will be created.
+	Region string `json:"region"`
+
+	// Zone specifies the IBM Cloud colo region where the cluster will be created.
+	// At this time, only single-zone clusters are supported.
+	Zone string `json:"zone"`
+
+	// VPCRegion specifies the IBM Cloud region in which to create VPC resources.
```

Comment:
What is the use case for having the VPC resources in a different region than the other cluster resources?

Reply:
are you asking why we have a Region and a VPCRegion? VPC Region is for the IBM Cloud region, e.g. `eu-gb`. Region is for the Power VS (different DCs and DC names) that is colocated with that IBM Cloud region, e.g. `lon`.